dbschema/progress-report.esdl

```diff
@@ -4,3 +4,24 @@ module default {
     overloaded engagement: LanguageEngagement;
   }
 }
+
+module ProgressReport {
+   abstract type Child extending Engagement::Child, Project::ContextAware {
+    annotation description := "\
+      A type that is a child of a progress report. \
+
+      It will always have a reference to a single progress report and engagement that it is under.";
+    
+    required progressReport: default::ProgressReport {
```

Comment:
Seems redundant, can we just call it report?
```suggestion
    required report: default::ProgressReport {
```
Matches 
https://github.com/SeedCompany/cord-api-v3/blob/develop/src/components/progress-report/variance-explanation/variance-explanation.dto.ts#L27-L27